packages/components/psammead-media-player/src/index.jsx

```diff
@@ -35,27 +36,42 @@ export const CanonicalMediaPlayer = ({
   return (
     <StyledContainer portrait={portrait}>
       {placeholderActive ? (
-        <Placeholder onClick={handlePlaceholderClick} src={placeholderSrc} />
+        <Placeholder
+          onClick={handlePlaceholderClick}
+          src={placeholderSrc}
+          srcset={placeholderSrcset}
+        />
       ) : (
         <Canonical src={src} />
       )}
     </StyledContainer>
   );
 };
 
-export const AmpMediaPlayer = ({ placeholderSrc, portrait, src, skin }) => {
+export const AmpMediaPlayer = ({
+  placeholderSrcset,
+  placeholderSrc,
+  portrait,
+  src,
+  skin,
+}) => {
   const StyledContainer =
     skin === 'audio' ? StyledAudioContainer : StyledVideoContainer;
 
   return (
     <StyledContainer portrait={portrait}>
-      <Amp placeholderSrc={placeholderSrc} src={src} />
+      <Amp
+        placeholderSrcset={placeholderSrcset}
+        placeholderSrc={placeholderSrc}
+        src={src}
+      />
     </StyledContainer>
   );
 };
 
 CanonicalMediaPlayer.propTypes = {
   placeholderSrc: string,
+  placeholderSrcset: string,
```

Comment:
Why is `placeholderSrc` and `placeholderSrcset` a required property in AmpMediaPlayer, but not CanonicalMediaPlayer?

Reply:
should be the case in both ends, fixing it